src/components/Item.css

```diff
@@ -0,0 +1,15 @@
+.soon {
+  color: #ad2d2d;
+}
+
+.kind-of-soon {
+  color: #f77903;
+}
+
+.not-so-soon {
+  color: #08640d;
+}
+
+.inactive {
+  color: lightgray;
+}
```

Comment:
I love the use of CSS to differentiate item groups, so we can still see the list together! 😁